### ISSUE TITLE
Group_Control_Border ignoring arguments

### ISSUE THREAD

**Issue by TinyGiantStudios** [elementor/elementor#11666]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When using https://developers.elementor.com/elementor-controls/border-control/ in a custom widget, all arguments except `name` and `selector` are **ignored**. This can be seen when using the code from that very docs page:

```
<?php
class Elementor_Test_Widget extends \Elementor\Widget_Base {

	protected function _register_controls() {

		$this->start_controls_section(
			'content_section',
			[
				'label' => __( 'Content', 'plugin-name' ),
				'tab' => \Elementor\Controls_Manager::TAB_CONTENT,
			]
		);

		$this->add_group_control(
			\Elementor\Group_Control_Border::get_type(),
			[
				'name' => 'border',
// LABEL FIELD IS IGNORED
				'label' => __( 'Border', 'plugin-domain' ),
				'selector' => '{{WRAPPER}} .wrapper',
			]
		);

		$this->end_controls_section();

	}

	protected function render() {
		$settings = $this->get_settings_for_display();
		echo '<div class="wrapper"> ... </div>';
	}

	protected function _content_template() {
		?>
		<div class="wrapper"> ... </div>
		<?php
	}

}
```

Here's what that looks like in Elementor:

![Markup on 2020-06-17 at 18:00:23](https://user-images.githubusercontent.com/685686/84927265-7c143880-b0c4-11ea-9575-f2781048cca0.png)

## Steps to reproduce

1. Create a new widget using code at https://developers.elementor.com/elementor-controls/border-control/
2. Add widget to a page
3. Notice that `'label' => __( 'Border', 'plugin-domain' ),` argument is ignored. The same happens for `description `, `show_label`, `label_block`.

This begs the question - is the documentation outdated or is this a bug?

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```

== Server Environment ==
	Operating System: Darwin
	Software: nginx/1.16.0
	MySQL version: MySQL Community Server - GPL v8.0.16
	PHP Version: 7.3.5
	PHP Max Input Vars: 4000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.4.2
	Site URL: http://wpjmtest.local
	Home URL: http://wpjmtest.local
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: ---
	Debug Mode: Active

== Theme ==
	Name: Storefront
	Version: 2.5.7
	Author: Automattic
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.9.12
		Author: Elementor.com

	Elementor Pro
		Version: 2.9.5
		Author: Elementor.com

	WP Job Manager
		Version: 1.34.2
		Author: Automattic

	WP Job Manager - Elementor Connector for Resumes
		Version: 1.0
		Author: TinyGiantStudios

	WP Job Manager - Resume Manager
		Version: 1.18.1
		Author: Automattic


== Must-Use Plugins ==
	Local by Flywheel Relative URL (for Live Links)
		Version: 1.1
		Author: Flywheel


== Elements Usage ==
	
	popup : 1
		heading : 1
		login : 1
	single : 4
		author-box : 1
		theme-post-content : 1
		theme-post-title : 1
	wp-post : 0
		shortcode : 1
	wp-page : 5
		heading : 1
		wprmc-resumes : 1


== Log ==
	


```
</details>

**Comment by rami-elementor** [comment]
Hi @TinyGiantStudios

Thank you for the feedback. Docs updated.

**Comment by nicholaszein** [comment]
Hello @TinyGiantStudios!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.9.0**! 🥳 

Thank you so much for your patience! 🙏 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
